gradle.properties

```
@@ -26,26 +26,26 @@ release_type=release
 # https://maven.parchmentmc.org/org/parchmentmc/data
 # Forge dependencies use maven's version range spec:
 # https://maven.apache.org/enforcer/enforcer-rules/versionRanges.html
-minecraft_version=1.21.7
-parchment_version=1.21.6-2025.06.29
+minecraft_version=1.21.8
+parchment_version=1.21.8-2025.07.20
 supported_mc_versions=
 
 fabric_loader_version=0.16.14
```

We could bump the loader too:

```suggestion
fabric_loader_version=0.17.2
```